extractor/src/main/java/org/schabi/newpipe/extractor/ServiceList.java

```diff
@@ -37,17 +37,19 @@ private ServiceList() {
 
     public static final YoutubeService YouTube;
     public static final SoundcloudService SoundCloud;
+    public static final PeertubeService PeerTube; 
     public static final MediaCCCService MediaCCC;
 
     /**
      * When creating a new service, put this service in the end of this list,
      * and give it the next free id.
      */
-    private static final List<StreamingService> SERVICES = unmodifiableList(
-            asList(
+    private static final List<StreamingService> SERVICES = Collections.unmodifiableList(
+            Arrays.asList(
                     YouTube = new YoutubeService(0),
                     SoundCloud = new SoundcloudService(1),
-                    MediaCCC = new MediaCCCService(2)
+                    PeerTube = new PeertubeService(2),
```

Comment:
This move is Illegal, as it will mess up the database. Please never change the order of the numbers.